hack/generate/samples/internal/ansible/advanced_molecule.go

```diff
@@ -0,0 +1,533 @@
+ && ansible-galaxy collection install /tmp/fixture_collection/operator_sdk-test_fixtures-0.0.0.tar.gz
+RUN echo abc123 > /opt/ansible/pwd.yml \
+ && ansible-vault encrypt_string --vault-password-file /opt/ansible/pwd.yml 'thisisatest' --name 'the_secret' > /opt/ansible/vars.yml
+`
+	err = kbtestutils.InsertCode(
+		filepath.Join(ma.ctx.Dir, "Dockerfile"),
+		"COPY playbooks/ ${HOME}/playbooks/",
+		dockerfileFragment)
+	pkg.CheckError("replacing Dockerfile", err)
+}
+
+func (ma *AdvancedMolecule) updatePlaybooks() {
+	log.Infof("adding playbook for argstest")
+	const argsPlaybook = `---
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.kubernetes
+  tasks:
+    - name: Get the decrypted message variable
+      include_vars:
+        file: /opt/ansible/vars.yml
+        name: the_secret
+    - name: Create configmap
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: '{{ meta.name }}'
+            namespace: '{{ meta.namespace }}'
+          data:
+            msg: The decrypted value is {{the_secret.the_secret}}
+`
+	err := testutils.ReplaceInFile(
+		filepath.Join(ma.ctx.Dir, "playbooks", "argstest.yml"),
+		originalPlaybookFragment,
+		argsPlaybook)
+	pkg.CheckError("adding playbook for argstest", err)
+
+	log.Infof("adding playbook for casetest")
+	const casePlaybook = `---
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.kubernetes
+  tasks:
+    - name: Create configmap
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: '{{ meta.name }}'
+            namespace: '{{ meta.namespace }}'
+          data:
+            shouldBeCamel: '{{ camelCaseVar | default("false") }}'
+`
+	err = testutils.ReplaceInFile(
+		filepath.Join(ma.ctx.Dir, "playbooks", "casetest.yml"),
+		originalPlaybookFragment,
+		casePlaybook)
+	pkg.CheckError("adding playbook for casetest", err)
+
+	log.Infof("adding playbook for inventorytest")
+	const inventoryPlaybook = `---
+- hosts: test
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: "inventorytest"
+
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - command: echo hello
+    - debug: msg='{{ "hello" | test }}'`
+	err = testutils.ReplaceInFile(
+		filepath.Join(ma.ctx.Dir, "playbooks", "inventorytest.yml"),
+		"---\n- hosts: localhost\n  gather_facts: no\n  collections:\n    - community.kubernetes\n    - operator_sdk.util\n  tasks:\n    - import_role:\n        name: \"inventorytest\"",
+		inventoryPlaybook)
+	pkg.CheckError("adding playbook for inventorytest", err)
+
+	log.Infof("adding playbook for reconciliationtest")
+	const reconciliationPlaybook = `---
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.kubernetes
+  tasks:
+    - name: retrieve configmap
+      k8s_info:
+        api_version: v1
+        kind: ConfigMap
+        namespace: '{{ meta.namespace }}'
```

Comment:
should this not be `ansible_operator_meta` like the other one? https://github.com/operator-framework/operator-sdk/blob/master/testdata/ansible/memcached-operator/roles/memcached/tasks/main.yml#L9

Reply:
No. Because we pass in the watches file the var meta. see; https://github.com/operator-framework/operator-sdk/blob/4d5b8839067a7f26155002fdd88f8fb7698becbe/hack/generate/samples/internal/ansible/testdata/watches.yaml#L57-L62

Reply:
hmm. IMO we ought to remove that and just access ansible_operator_meta directly. That said, it does not need to be in this PR, just something I noticed.